seatunnel-core/seatunnel-core-flink-sql/src/main/bin/start-seatunnel-sql.sh

```diff
@@ -32,19 +32,24 @@ else
     args=$@
 fi
 
-ENV_PARAMETERS=$(java -cp ${APP_JAR} org.apache.seatunnel.core.sql.FlinkEnvParameterParser ${args})
-echo "Export JVM_ARGS: ${ENV_PARAMETERS}"
-export JVM_ARGS="${ENV_PARAMETERS}"
+ENV_PARAMETERS_OR_ERROR=$(java -cp ${APP_JAR} org.apache.seatunnel.core.flink.FlinkEnvParameterParser ${args}) && EXIT_CODE=$? || EXIT_CODE=$?
+if [ ${EXIT_CODE} -eq 0 ]; then
+  echo "Export JVM_ARGS: ${ENV_PARAMETERS_OR_ERROR}"
+  export JVM_ARGS="${ENV_PARAMETERS_OR_ERROR}"
+else
+    echo "${ENV_PARAMETERS_OR_ERROR}"
+    exit ${EXIT_CODE}
+fi
```

Comment:
```suggestion

```
In flink sql conf, we don't have the `env` lable in flink.sql.conf, so the `FlinkEnvParameterParser` will throw exception. We can delete this step in flink sql, we don't need to export env args.